src/promptflow-evals/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## v0.3.3 (Upcoming)
 ### Features Added
+- Add a new evaluator and associated adversarial content simulator for protected materials, which determines if given inputs contain materials protected by IP laws.
+- Added a new internal evaluator and associated adversarial simulator that's only available for internal use - the ECI eval/sim.

```

Comment:
If it's only for internal use, do we want to mention it in the CHANGELOG?

Reply:
Good point, removing.